studies/templates/studies/study_attachments.html

```diff
@@ -95,16 +95,16 @@ <h1>Videos</h1>
                         </tr>
                         </thead>
                         <tbody>
-                        {% for video in videos %}
+                        {% for video in video_list %}
                             <tr>
                                 <td><span class="small">{{ video.full_name }} </span></td>
                                 {% timezone request.user.time_zone %}
-                                    <td> {{ video.last_modified }} </td>
+                                    <td> {{ video.created_at }} </td>
                                 {% endtimezone %}
                                 <td>
-                                    <form method="POST"> {% csrf_token %}
-                                        <button name="attachment" type="submit" value="{{ video.download_url }}"
-                                                class="btn btn-primary btn-sm">Download
+                                    <form method="POST" action="{% url 'exp:study-response-video-download' pk=study.id video=video.pk%}"> {% csrf_token %}
```

Comment:
I left a note at the view, but this isn't RESTful - you're trying to fetch resources from the server; GET is appropriate here. Either use an anchor tag or method="GET"

Reply:
changed to GET here and in study_responses.html

Reply:
actually changed to using anchor tags